posthog/models/organization.py

```diff
@@ -155,9 +155,10 @@ def validate_update(
 
 class OrganizationInvite(UUIDModel):
     organization: models.ForeignKey = models.ForeignKey(
-        "posthog.Organization", on_delete=models.CASCADE, related_name="invites", related_query_name="invite"
+        "posthog.Organization", on_delete=models.CASCADE, related_name="invites", related_query_name="invite",
     )
     target_email: models.EmailField = models.EmailField(null=True, db_index=True)
+    first_name: models.CharField = models.CharField(max_length=30, blank=True, default="")
```

Comment:
Field name is `first_name`, but is this actually intended for first name explicitly? If this is only a holdover from `User.first_name` (which is kind of a legacy issue), then it should probably be just `name`

Reply:
Well it is indeed to match `User.first_name`, but I thought we were just using first name, not full name everywhere else too (e.g. the signup form says first name and the placeholder says "Jane"). So `first_name` sounds appropriate even then, thoughts?

Reply:
Hmm, I'm pretty sure we're using full names, just using a single field, as we don't use `User.last_name` _anywhere_. 🤔

Reply:
Hm, actually we are kind of using it… accidentally, as social auth fills it in on registration. We should get this straight either way. Thoughts @paolodamico?

Reply:
Well I'm not sure what most users did before we revamped the signup page, but for recent users not doing social auth, I think it's pretty straightforward to just type your first name. I would just do first name everywhere, it has easier backwards support, it's nicer to receive emails addressed like "Hey John! instead of "Hey John Doe!" and I don't think we really need the last name for anything.

The only reason I can think for this is for very large teams, but in that case maybe the email address is enough, or they could just do full name by their own decision?